tests/tips/test_feed_suggestion.py

```diff
@@ -22,16 +22,21 @@ async def test_no_tips(autopay_contract_setup, caplog):
 @pytest.mark.asyncio
 async def test_funded_feeds_only(setup_datafeed, caplog):
     """Test feed tips but no one time tips and no reported timestamps"""
-    datafeed, tip = await get_feed_and_tip(await setup_datafeed)
+    flex = await setup_datafeed
+    datafeed, tip = await get_feed_and_tip(flex.autopay)
     assert isinstance(datafeed, DataFeed)
     assert isinstance(tip, int)
+    assert tip == int(1e18)
     assert "No one time tip funded queries available" in caplog.text
+    # tests are failing when ran together
```

Comment:
This still happening?

Reply:
yeah and can't figure out why. the fixtures are function scoped so it shouldn't but  I'm not an expert on pytests so maybe I'm missing something and I also looked into the auto_use param in the fixtures and that doesn't fix it.